src/app/lib/styledGrid/index.jsx

```diff
@@ -27,6 +29,160 @@ import {
   gridContainerLargeCss,
 } from '../layoutGrid';
 
+export const MediumContentContainer = styled.div`
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    margin: 0 auto;
+    max-width: 46.4rem;
+  }
+`;
+
+const defaultColumns = {
+  group0: 6,
+  group1: 6,
+  group2: 6,
+  group3: 6,
+  group4: 6,
+  group5: 12,
+};
+
+const mediumColumns = {
+  ...defaultColumns,
+  group3: 5,
+  group4: 5,
+  group5: 10,
+};
+
+export const MediumGridWrapper = ({ children }) => {
```

Comment:
Can I suggest we follow the same naming convention to other grid components and name the size as a suffix rather than a prefix? E.G. `MediumGridWrapper` to `GridWrapperMedium`.

Reply:
That sounds good. 👍

Reply:
Done here https://github.com/bbc/simorgh/pull/6618/commits/572176cacd94d8497fc1afaa7296ef5de70729b3